ote/src/cljc/ote/transit_changes.cljc

```diff
@@ -129,10 +129,11 @@
 (defn earliest-departure-time [stop]
   (let [minutes-from-midnight1 (some-> stop :gtfs/departure-time-date1 (time/minutes-from-midnight))
         minutes-from-midnight2 (some-> stop :gtfs/departure-time-date2 (time/minutes-from-midnight))]
+    ; Use date2 departure time as master when sorting results. If both are nil set time to zero
```

Comment:
Why commenting putting a comment here is excellent, 
generally comments should first and foremost answer the question "Why?".
No need to remove this comment of course :)